### PR TITLE
User registration

### PR DESCRIPTION
**Pull request by mariomelchor**

### CODE REVIEW

File: client/src/pages/registration/Registration.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import Button from 'material-ui/Button';
+import TextField from 'material-ui/TextField';
+
+class Registration extends Component {
+  // Setting the initial values
+  state = {
+    username: "",
+    password: "",
+    firstName: "",
+    LastName: ""
```
Comment (saloiofun) [inline client/src/pages/registration/Registration.js]: Change to lower case "L" on "LastName"

File: client/src/pages/registration/Registration.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import React, { Component } from "react";
+import API from "../../utils/API";
+import Card, { CardContent } from 'material-ui/Card';
+import Button from 'material-ui/Button';
+import TextField from 'material-ui/TextField';
+
```
Comment (saloiofun) [inline client/src/pages/registration/Registration.js]: Use single quotation and remove semicolons

File: client/src/pages/registration/Registration.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  // handle any changes to the input fields
+  handleInputChange = event => {
+    const { name, value } = event.target;
+    // Set the state for the appropriate input field
+    this.setState({
+      [name]: value
+    });
+  };
+
```
Comment (saloiofun) [inline client/src/pages/registration/Registration.js]: Remove semicolons

File: client/src/pages/registration/Registration.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  // When the form is submitted
+  handleFormSubmit = event => {
+    event.preventDefault();
```
Comment (saloiofun) [inline client/src/pages/registration/Registration.js]: Remove semicolons

File: client/src/pages/registration/Registration.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+    API.registerUser({
+      email: this.state.username,
+      password: this.state.password,
+      firstName: this.state.firstName,
+      lastName: this.state.lastName
+    })
+    .then(res => {
+      if (res.data){
+        console.log(res.data);
```
Comment (saloiofun) [inline client/src/pages/registration/Registration.js]: Remove semicolons

File: client/src/pages/registration/Registration.js [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+      }
+    })
+    .catch(err => console.log(err))
+  };
```
Comment (saloiofun) [inline client/src/pages/registration/Registration.js]: Remove semicolons

File: client/src/pages/registration/Registration.js [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+              <Button raised color="primary" type="submit">Register</Button>
+            </form>
+          </CardContent>
+        </Card>
+      </div>
+    );
+  }
+}
+
+export default Registration;
```
Comment (saloiofun) [inline client/src/pages/registration/Registration.js]: Remove semicolons

File: client/src/App.js [hunk 8]
```diff
@@ -18,6 +18,7 @@ import Events from './pages/events'
 import GuestList from './pages/guestList'
 import SendInvites from './pages/sendInvites'
 import { DashboardListItems, EventsListItems, GuestListItems, SendInvitesListItems } from './tileData';
```
Comment (saloiofun) [inline client/src/App.js]: Remove semicolon

**Review by saloiofun — CHANGES_REQUESTED** [review]
Please verify

**Comment by mariomelchor** [comment]
Please review corrections

**Review by saloiofun — APPROVED** [review]

**Pull request merged**